server/testutils/factories/prisonAddress.ts

```diff
@@ -1,19 +1,14 @@
 import { faker } from '@faker-js/faker/locale/en_GB'
 import { Factory } from 'fishery'
 
-import type { PrisonAddress } from '@prison-api'
+import type { PrisonAddress } from '@prison-register-api'
 
-export default Factory.define<PrisonAddress>(({ params }) => {
-  const locality = params.locality || faker.location.county()
-  const premise = `HMP ${locality}`
-
-  return {
-    premise,
-    street: faker.location.streetAddress(),
-    locality,
-    town: faker.location.city(),
-    postalCode: faker.location.zipCode(),
-    country: 'United Kingdom',
-    primary: true,
-  }
-})
+export default Factory.define<PrisonAddress>(() => ({
+  id: faker.number.int({ min: 1 }),
+  addressLine1: faker.helpers.arrayElement([null, faker.location.streetAddress()]),
```

Comment:
My message about addresses above applies here too - let's remove the `null`ness from these factories apart from when setting `addressLine2` as that feels like a more real-world scenario.